x/ccv/consumer/keeper/relay.go

```diff
@@ -25,8 +25,10 @@ func (k Keeper) OnRecvVSCPacket(ctx sdk.Context, packet channeltypes.Packet, new
 	// get the provider channel
 	providerChannel, found := k.GetProviderChannel(ctx)
 	if found && providerChannel != packet.DestinationChannel {
-		// VSC packet was sent on a channel different than the provider channel
-		return utils.OnRecvPacketOnUnknownChannel(ctx, k.scopedKeeper, k.channelKeeper, packet)
+		// VSC packet was sent on a channel different than the provider channel;
+		// this should never happen
+		panic(fmt.Errorf("VSCPacket received on unknown channel %s; expected: %s",
+			packet.DestinationChannel, providerChannel))
```

Comment:
I'm not sure I understand why this is a panic. From reading https://github.com/cosmos/interchain-security/issues/292 I understand that the consumer can receive packets from channels which arent OPEN on the provider. It sounds like recoverable behavior, not panic behavior.